spec/ovirt_metrics_spec.rb

```diff
@@ -57,11 +87,6 @@ def assert_object_with_empty_samples_data(type)
 
   end
 
-  context "RHEV 3.0" do
```

Comment:
We do not support 30 any more, need to remove this from everywhere but this is for a follow-up pr. only removed from the test that it breaks.